mongo/integration/server_selection_prose_test.go

```diff
@@ -35,8 +108,8 @@ func TestServerSelectionProse(t *testing.T) {
 		hosts := options.Client().ApplyURI(mtest.ClusterURI()).Hosts
 		require.GreaterOrEqualf(mt, len(hosts), 2, "test cluster must have at least 2 mongos hosts")
 
-		// Set a failpoint on a specific mongos host that delays all "find" commands for 500ms. We
-		// need to know which mongos we set the failpoint on for our assertions later.
+		// Set a failpoint on a specific mongos host that delays all "find" commands for 500ms. We need to know which
+		// mongos we set the failpoint on for our assertions later.
```

Comment:
Do we have a specific character count we like to keep comments to?  If not is it okay if I keep them at 120?